virttest/utils_misc.py

```diff
@@ -2281,7 +2281,7 @@ def _get_backend_dir(params):
     Get the appropriate backend directory. Example: backends/qemu.
     """
     return os.path.join(data_dir.get_root_dir(), 'backends',
-                        params.get("vm_type"))
+                        params.get("vm_type", ""))
```

Comment:
A small improvement here is not in the documentation that the base directory can be returned with the empty parameter.